addons/mail/wizard/mail_compose_message.py

```diff
@@ -592,7 +592,7 @@ def render_message(self, res_ids):
             res_ids = [res_ids]
 
         subjects = self._render_field('subject', res_ids, options={"render_safe": True})
-        bodies = self._render_field('body', res_ids, post_process=True)
+        bodies = self._render_field('body', res_ids, post_process=True, options={'preserve_comments': self.composition_mode == 'mass_mail'})
```

Comment:
Why here, what is the purpose of this code (at least a comment in code maybe) ? Why not in template send mail (generate_email) ? Why not in digest, as we also have some specific styling in digests emails (not sure if comments, but globally it is the same behavior) ?

Reply:
I think we could probably be defensive and do the same when rendering digest main layout as it contains manual styling (a bit like mass mailing styling added in marketing emails). That way we are already comments-compliant :) . Could you have a quick look maybe ?

Reply:
Here, it's done. Hope that's what you meant.

Reply:
Indeed, code in digest seems correct :) thanks